spec/factories/user_profiles.rb

```diff
@@ -0,0 +1,5 @@
+FactoryGirl.define do
+  factory :user_profile do
+    bio "MyString"
```

Comment:
Since `bio` is not a required field, it may be better to leave it out of the factory definition (so that it would default to `nil`) and including it via `create(:user_profile, bio: 'foo')` when an initial value is needed.